eclass/kernel-install.eclass

```diff
@@ -508,6 +666,18 @@ kernel-install_pkg_postinst() {
 	if [[ -z ${ROOT} ]]; then
 		kernel-install_install_all "${dir_ver}"
 	fi
+
+	if [[ ${KERNEL_IUSE_GENERIC_UKI} ]]; then
+		if use generic-uki; then
+			ewarn "The prebuilt initramfs and unified kernel image are highly experimental!"
+			ewarn "These images may not work on your system. Please ensure that a working"
+			ewarn "alternative kernel(+initramfs) or UKI is also installed before rebooting!"
+			ewarn "Note that when secureboot is enabled in the firmware settings any kernel"
+			ewarn "command line arguments supplied to the UKI by the bootloader are ignored"
+			ewarn "To ensure the root partition can be found, systemd-gpt-auto-generator must"
+			ewarn "be used. See the systemd-gpt-auto-generator manual for more information."
```

Comment:
This manpage is not very friendly. Do we have a Wiki page for how to set the system up for this?

Reply:
> This manpage is not very friendly. Do we have a Wiki page for how to set the system up for this?

Not yet, but I will definitely write one later.

Reply:
> > This manpage is not very friendly. Do we have a Wiki page for how to set the system up for this?
> 
> Not yet, but I will definitely write one later.

Done: https://wiki.gentoo.org/wiki/Systemd#Automatic_mounting_of_partitions_at_boot